BUILD

```diff
@@ -17,37 +17,7 @@ tf_cc_shared_object(
     ],
     visibility = ["//visibility:public"],
     deps = [
-        "//third_party/xla_client:runtime",
```

Comment:
curious why can we delete these xla_client dependency, are we getting it from somewhere else or these are not needed for the `XLAC.so`?

Reply:
We needed them before migrating `csrc` to bazel, since code in there depended on `libcomputation_client.so`. Now that everything is in bazel, bazel willnclude all of the transitive deps of `init_python_bindings` on its own.